src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -229,6 +229,9 @@
   <data name="Arg_CannotHaveNegativeValue" xml:space="preserve">
     <value>String cannot contain a minus sign if the base is not 10.</value>
   </data>
+  <data name="Arg_CannotMarshal" xml:space="preserve">
```

Comment:
Can we replace a native resource message with this? If so, we should file an issue to track that work.

Reply:
I don't think there's a native parser for the managed resource string format - this would have to call managed code from CoreCLR. I don't know how feasible that is in interop code generation. Sounds like a lot of work for questionable benefit? Unless I'm mistaken and this is actually easy.